dubbo-demo/dubbo-demo-api/dubbo-demo-api-consumer/pom.xml

```diff
@@ -40,7 +40,11 @@
         </dependency>
         <dependency>
             <groupId>org.apache.dubbo</groupId>
-            <artifactId>dubbo-registry-multicast</artifactId>
```

Comment:
Please do not remove the the multicast dependency.  We need to keep the convenience for People to switch between multicast and zookeeper.